From: Kleber Sacilotto de Souza <klebers@linux.vnet.ibm.com>
Date: Thu, 10 Jul 2014 10:17:08 -0300
Subject: [PATCH] GenWQE: Remove unnecessary include
Patch-mainline: v3.17-rc1
Git-commit: 5b35b20d4eb4c7fa6263939304c69207f791535f
References: bsc#949731,LTC#131502

The include for the UAPI header file from card_base.c can be removed
since it's already included on card_base.h.

Signed-off-by: Kleber Sacilotto de Souza <klebers@linux.vnet.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: John Jolly <jjolly@suse.de>
---
 drivers/misc/genwqe/card_base.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/misc/genwqe/card_base.c
+++ b/drivers/misc/genwqe/card_base.c
@@ -38,7 +38,6 @@
 #include <linux/notifier.h>
 #include <linux/device.h>
 #include <linux/log2.h>
-#include <linux/genwqe/genwqe_card.h>
 
 #include "card_base.h"
 #include "card_ddcb.h"
